### ISSUE TITLE
Change project name

### ISSUE THREAD

**Issue by mcanevet** [camptocamp/conplicity#88]

**Comment by mcanevet** [comment]
Free Open Network Data Universal Engine

**Comment by saimonn** [comment]
I have an idea. see #67 
Tip: backup + docker = baker

**Comment by mcanevet** [comment]
Or docker + backup = dockup

**Comment by saimonn** [comment]
I beg you to draw a dockup ;)

**Comment by raphink** [comment]
AirFreight: taking your container data to the cloud

**Comment by raphink** [comment]
Or AirCargo

**Comment by mcanevet** [comment]
Or Dahu, the problem is to find an acronym to match this...

**Comment by mcanevet** [comment]
or Bivouac : Backup Volume OrchestrAtor

**Comment by madchap** [comment]
clodo : CLOne DOcker

/me sors.

**Comment by saimonn** [comment]
AVOCADO  Another VOlume Clone and bAckup tool for  DOcker

(inspiration from http://acronymcreator.net/ace.py )

**Comment by mcanevet** [comment]
@saimonn Avocado is already an opensource project : https://github.com/avocado-framework/avocado

**Comment by saimonn** [comment]
BACON | BAckup CONtainers  ?

**Comment by mcanevet** [comment]
@saimonn I like bacon but : https://github.com/baconjs/bacon.js/

**Comment by mcanevet** [comment]
@saimonn I think you should remove "clone" from your acronym generator.
"cloning" a container does not make me think about backup.

**Comment by saimonn** [comment]
there's still Bender on the list then :) And if we choose that, we already have a logo.

**Comment by saimonn** [comment]
**Codaba** | COntainer DAta BAckup

**Covoba** | COntainer VOlume Backup

**Dodaba** | DOcker DAta Backup

**Dovoba** | DOcker VOlume Backup

**Comment by blaggacao** [comment]
https://en.wiktionary.org/wiki/bivouac#Etymology - that fits. I wouldn't use french writing though, most people might get confused with the pronunciation...
Bywatch, your second guard if the enemy strikes...

It was re-borrowed into german "biwak" from french, evolving it into "biwakt" (with a "t") would give syntactical allusion to the german form of germanic "-ed" suffix, resulting in a etymlogically derived syntactic meaning of: "guarded" or a modern german translation "bewacht".

Overall that's goals aligned, I think.
Also an option: "biwaked" but people might pronounce "bi" and "waked" like in "awake". Which interestingly enough has the same ethymlogocial root as biwak itself 😄

**Comment by raphink** [comment]
@blaggacao Yes... I'm not a huge fan of the German spelling, because it's not obvious how to pronounce it (I know I always have issues with Augeas, whose German inceptor pronounces with a hard "g" but I tend to pronounce with a soft one).

There is an alternative spelling used in Spanish and formerly in French, which is "bivac". That could stand for Backup Interface for Volumes Attached to Containers. There is one project on GitHub with that name, but it's totally unrelated.

The logo could be something like

![img_1880](https://user-images.githubusercontent.com/650430/38475083-eef9bfcc-3b6a-11e8-9948-466c8b64c572.PNG)

**Comment by raphink** [comment]
https://fr.wiktionary.org/wiki/bivouac explains how the word "bivouac" came to the French language. From German "biwacht" (< bewachen, to keep guard), **Swiss mercenaries** imported the word into French as **bivac**. From there, it made its way into other European languages through the Napoleonian wars.

So according to this description, it's the Swiss that made the change from biwacht to bivac.

**Comment by raphink** [comment]
Or using the 20 DB font, whose capital A looks like a tent:

![bivac](https://user-images.githubusercontent.com/650430/38479400-8abfaa34-3b84-11e8-9fc5-b0b64427adf1.png)

![bivac_logo](https://user-images.githubusercontent.com/650430/38479401-8e379564-3b84-11e8-96e9-8d13cab5f814.png)

**Comment by raphink** [comment]
@mcanevet @saimonn for the context, @blaggacao and I discussed this subject this morning over breakfast 😄

**Comment by raphink** [comment]
Reusing the A for the tent logo, to make it more in the same style as the text

![bivac_tent3_conts_dark](https://user-images.githubusercontent.com/650430/38505808-b4f8cc0a-3bdd-11e8-98bc-2671ce3200d3.png)

**Comment by raphink** [comment]
![bivac_tent3_cont_orange](https://user-images.githubusercontent.com/650430/38510601-2844def8-3beb-11e8-867b-466fa8d15a57.png)

![bivac_tent3_cont_dark2](https://user-images.githubusercontent.com/650430/38510608-2d606056-3beb-11e8-961c-d8353b1118c6.png)

**Comment by raphink** [comment]
Fixing the perspective of the front of the tent:

![bivac_tent3_cont_orange_green_front](https://user-images.githubusercontent.com/650430/38527563-042e8c68-3c21-11e8-9891-f21677afbd34.png)

**Comment by raphink** [comment]
With a background colour in the tent (to avoid transparency effects) and perspective on the box

![bivac_tent3_cont_orange_green3_front_persp](https://user-images.githubusercontent.com/650430/38527988-4154f29c-3c23-11e8-8e57-9690d821cea8.png)

**Comment by blaggacao** [comment]
Looks pretty nice slaready, here are my 2 cents:
- Green: nice, secondary colours?
- I would use a sleeker font (eg. Lato, or try Comfortaa)
- I wouldn't use the acronym (Doesn't make it more professional in my perception)

**Comment by raphink** [comment]
Thanks for the feedback @blaggacao.

* there's green and orange
* if you've read the thread, the font was chosen specifically because the A looks like a tent. Changing the font makes the whole design kind of moot
* I added the acronym to make a full version, with the idea that I can also make versions without the acronym, or even with the tent only

**Comment by blaggacao** [comment]
> if you've read the thread, the font was chosen specifically because the A looks like a tent. Changing the font makes the whole design kind of moot

I know it's just not as sleek as it could be. It's more an "eye" thing... But hey, on the other hand, why not try something new in software logo design? :smile: :

**Comment by raphink** [comment]
By sleek, you mean sans-serif + light weight? Or just that using a retro font for software is not common?

**Comment by raphink** [comment]
> But hey, on the other hand, why not try something new in software logo design? 😄

At least the logo will be easily recognisable with a different kind of font

**Comment by blaggacao** [comment]
> By sleek, you mean sans-serif + light weight? Or just that using a retro font for software is not common?

Exactly. Both: yes.
sans-serif + bold is also ok :+1: 

But I neither want o impose my aesthetics :smile:

**Comment by mcanevet** [comment]
Let's go wih `Bivac`

**Comment by mcanevet** [comment]
Finally \o/
